src/caret-down-12px.svg

```diff
@@ -0,0 +1,3 @@
+<svg width="12" height="12" viewBox="0 0 12 12" fill="none" xmlns="http://www.w3.org/2000/svg">
+    <path d="M2.64192 4.8334C2.32962 4.51695 2.56651 4 3.02381 4H8.97618C9.434 4 9.67068 4.51794 9.35735 4.83413L6.37509 7.84371C6.1683 8.05239 5.81837 8.05205 5.61203 7.84297L2.64192 4.8334Z" fill="#818594"/>
```

Comment:
please remove the `fill` attributes from both `svg` and `path` elements. We apply `fill: currentcolor` in Ring UI CSS